src/NSwag.AspNetCore/NSwag.AspNetCore.csproj

```diff
@@ -63,6 +68,10 @@
     <ProjectReference Include="..\NSwag.Generation\NSwag.Generation.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This probably needs to be removed, no?

Reply:
You are right, removed.

Sorry for the delay, I was out of the keyboard.